### PR TITLE
Corrections in Horizon Europe catalog

### PR DESCRIPTION
**Pull request by torsten-rathmann**
Some issues can only be seen if you read the whole in silence... 

I have found a number of issues in our offer for Horizon Europe projects. In this pull request are my recommendations concerning the catalog horizon-europe.xml. The most important alteration is presumably the replacement of a wrongly cited original question (No. 7). The numbering follows the sequence of questions in the catalog.

1. Typing error correction in help text for question set horizon-europe/general/general "Basic information on the project": 

"In addition, for many questions there is the option of answering the question on a dataset-specific basis."

instead of

"In addition, for many questions there is the option of answering the question on a database-specific basis."

2. Replacement of example in help text for question horizon-europe/data_summary/type/data_kind "What kind of dataset is it?"

"scanning electron micrographs" // "rasterelektronenmikroskopische Aufnahmen"

instead of

"software developed within the project" // "Software, die im Projekt entwickelt wird"

Reason: European Commission (EC) classifies software into "other research output", not into research data.

3. Text area for question horizon-europe/data_summary/provenance/provenance "What is the provenance of the data?"

Reason: We don't really know which parts of data provenance are important for research data management. A description of data provenance may be detailled and needs space for longer texts. Additionally, the help text doesn't match the option set that is used now.

4. Typing error correction in help text for question horizon-europe/data_summary/data_utility/reuse_scenario "Which individuals, groups or institutions could be interested in re-using this dataset? What are possible scenarios?"

"such as privacy" instead of "such as Privacy"

5. Shortening question text for question horizon-europe/data_summary/data_utility/reuse_scenario "Which individuals, groups or institutions could be interested in re-using this dataset? What are possible scenarios?"

Deleting "What are possible scenarios?" // "Welche Konsequenzen hat das Nachnutzungspotential später für die Bereitstellung der Daten?" // "Quels sont les scénarios possibles??" // "Quali sono gli scenari possibili?"

Reason: The deleted second question doesn't match the option set, which is used for this question because entering longer texts is very uncomfortable. The second question is also not of importance for EC.

6. Deletion of help in question horizon-europe/fair_data/accessible_data/access_restrictions_explanation "If there are any restrictions on the re-use of this dataset, please explain why."

Deleted: "One possible description" (above textarea for answer)

Reason: Help text unnecessary in this manner. Hidden link cannot be clicked and the link target, a Welcome Trust document, contains very general information.

7. Wrong original question replaced in questionset horizon-europe/fair_data/accessible_metadata Accessibility: metadata

"Will documentation or reference about any software be needed to access or read the data be included? Will it be possible to include the relevant software (e.g. in open source code)?"

instead of 

"If an embargo is applied to give time to publish or seek protection of the intellectual property (e.g. patents), specify why and how long this will apply, bearing in mind that research data should be made available as soon as possible."

8. Wording in translation in question horizon-europe/fair_data/interoperable_data/qualified_references "Will your data include qualified references to other data (e.g. other data from your project, or datasets from previous research)?"

"qualifizierte Referenzen" instead of "geeignete Verweise"

9. Correction of translation in question "How long will the data be stored?"

"Wie lange werden die Daten aufbewahrt werden?"

instead of

"Wie lange müssen die Daten aufbewahrt werden?"

10. Typing error correction in help text for question horizon-europe/resources/selection/preservation__criteria "What are the criteria / rules for the selection of the data to be archived (after the end of the project)?"

"gesellschaftliche oder politische Relevanz"

instead of

"gesellschaftliche oder politisch Relevanz"

11. Typing error correction in help text for question horizon-europe/ethics/intellectual-property-rights-yesno/yesno "Does the project use and/or produce data that is protected by intellectual or industrial property rights?"

"Sammelwerke" instead of "Sammelbankwerke"

12. Enable condition in questionset horizon-europe/ethics/intellectual-property-rights-dataset "Intellectual property rights II"

13. Typing error correction in questionset horizon-europe/rdm_regulations/project-partners-partner

"Project partners" instead of "project partners"

14. Typing error correction in question horizon-europe/rdm_regulations/project-partners-partner/name

"Project partner" instead of "project partner"

15. Typing error correction in help text for question horizon-europe/rdm_regulations/project-partners-partner/rdm_policy "Does your institution have rules or guidelines for the handling of research data? If yes, please briefly outline them and refer to more detailed sources of information if necessary. Please also indicate, if the rules / guidelines are mandatory or optional."

"forschungsdaten.org" instead of "Forschungsdaten.org"

### CODE REVIEW

File: rdmorganiser/questions/horizon-europe.xml [hunk 1]
```diff
@@ -2385,8 +2380,8 @@ A qualified reference is a cross-reference that explains its intent. For example
 		<default_text lang="en"/>
 		<verbose_name lang="en"/>
 		<verbose_name_plural lang="en"/>
-		<help lang="de">Ein qualifizierter Verweis ist ein Querverweis, der seine Absicht erklärt. Zum Beispiel ist "X ist Regulator von Y" eine viel qualifiziertere Referenz als "X ist mit Y verbunden", oder "X siehe auch Y". Ziel ist es daher, möglichst viele sinnvolle Verknüpfungen zwischen (Meta-)Datenressourcen zu schaffen, um das Kontextwissen über die Daten zu bereichern.</help>
-		<text lang="de">Werden Ihre Daten geeignete Verweise auf andere Daten enthalten (z. B. auf andere Daten aus Ihrem Projekt oder auf Datensätze aus früheren Forschungsarbeiten)?</text>
+		<help lang="de">Ein qualifizierte Referenz ist ein Querverweis, der seine Absicht erklärt. Zum Beispiel ist "X ist Regulator von Y" eine viel qualifiziertere Referenz als "X ist mit Y verbunden", oder "X siehe auch Y". Ziel ist es daher, möglichst viele sinnvolle Verknüpfungen zwischen (Meta-)Datenressourcen zu schaffen, um das Kontextwissen über die Daten zu bereichern.</help>
```
Comment (Zack-83) [inline rdmorganiser/questions/horizon-europe.xml:2383]:
```suggestion
		<help lang="de">Eine qualifizierte Referenz ist ein Querverweis, der seine Absicht erklärt. Zum Beispiel ist "X ist Regulator von Y" eine viel qualifiziertere Referenz als "X ist mit Y verbunden", oder "X siehe auch Y". Ziel ist es daher, möglichst viele sinnvolle Verknüpfungen zwischen (Meta-)Datenressourcen zu schaffen, um das Kontextwissen über die Daten zu bereichern.</help>
```

**Review by Zack-83 — APPROVED** [review]

**Review by Zack-83 — CHANGES_REQUESTED** [review]
undo all

**Review by Zack-83 — APPROVED** [review]
Ich bin zu einer Neuformulierung leider nicht mehr dazu gekommen. Wenn es eilt, können wir mergen

**Comment by torsten-rathmann** [comment]
Mergen wäre gut, weil dann als Nächstes die Korrekturen in den Optionen als Pull Request angeboten werden können. Ich habe aber selbst nicht das Recht, diesen Beitrag zu mergen. Wer kann helfen?

**Pull request merged**